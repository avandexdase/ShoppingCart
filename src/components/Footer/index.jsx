import React from 'react';

export default function Footer() {
  return (
    <footer>
      <div className="main-footer">
        <span>
          Copyright &#169; 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd
        </span>
      </div>
    </footer>
  );
}
