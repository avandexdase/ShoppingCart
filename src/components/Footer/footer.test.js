import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './index';

it('<footer />', () => {
  const component = renderer.create(
    <Footer />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});