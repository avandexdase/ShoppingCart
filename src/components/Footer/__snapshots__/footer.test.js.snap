// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<footer /> 1`] = `
<footer>
  <div
    className="main-footer"
  >
    <span>
      Copyright Â© 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd
    </span>
  </div>
</footer>
`;
